src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <RootNamespace>System.Collections.NonGeneric.Tests</RootNamespace>
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```

Comment:
It also appears to require a VS option that's off by default (it was for me in VS 2015 Enterprise).  I had to enable Universal Windows App Development Tools in order to build.